intellij-ext/build.gradle.kts

```diff
@@ -69,24 +69,29 @@ tasks {
 
         // Extract the <!-- Plugin description --> section from README.md and provide for the plugin's manifest
         pluginDescription.set(
-                projectDir.resolve("README.md").readText().lines().run {
-                    val start = "<!-- Plugin description -->"
-                    val end = "<!-- Plugin description end -->"
+            projectDir.resolve("README.md").readText().lines().run {
+                val start = "<!-- Plugin description -->"
+                val end = "<!-- Plugin description end -->"
 
-                    if (!containsAll(listOf(start, end))) {
-                        throw GradleException("Plugin description section not found in README.md:\n$start ... $end")
-                    }
-                    subList(indexOf(start) + 1, indexOf(end))
-                }.joinToString("\n").run { markdownToHTML(this) }
+                if (!containsAll(listOf(start, end))) {
+                    throw GradleException("Plugin description section not found in README.md:\n$start ... $end")
+                }
+                subList(indexOf(start) + 1, indexOf(end))
+            }.joinToString("\n").run { markdownToHTML(this) }
         )
     }
 
     prepareSandbox {
-        from(file("$projectDir/libmirrord_layer.dylib")) {
-            into(pluginName.get())
-        }
-        from(file("$projectDir/libmirrord_layer.so")) {
-            into(pluginName.get())
+        val sharedLibs =
+            mapOf("macos" to "$projectDir/libmirrord_layer.dylib", "linux" to "$projectDir/libmirrord_layer.so")
+        sharedLibs.forEach { (_, lib) ->
+            from(file(lib)) {
+                into(pluginName.get())
+            }
+            // NOTE: comment this line when developing locally without either of shared libs
+            if (!System.getenv("CI_BUILD_PLUGIN").toBoolean()) {
```

Comment:
The VS Code extension has the "vscode.ExtensionMode.Development" flag, which then lets us grab the lib from target/debug instead of the cwd. Anything similar in IntelliJ?

Reply:
hmm not sure, the main reason for adding thsi exception was that we shipped a version of mirrord without even loading the dylib, so if in the CI it fails, we know its an issue with dylibs not being put in the plugin dir